packages/flutter/test/widgets/two_dimensional_viewport_test.dart

```diff
@@ -161,10 +161,18 @@ void main() {
         ));
         await tester.pumpAndSettle();
 
+        // In the tests below the number of RepaintBoundary widgets depends on:
+        // TwoDimensionalChildListDelegate - builds 1 unless addRepaintBoundaries is false
+        // ModalRoute - builds 2
+        // GlowingOverscrollIndicator - builds 2
+        // RawScrollbar - builds 2
+
         switch (defaultTargetPlatform) {
-          case TargetPlatform.android:
           case TargetPlatform.fuchsia:
+            // Includes a scrollbar and an overscroll glow
             expect(find.byType(RepaintBoundary), findsNWidgets(7));
+
+          case TargetPlatform.android:
```

Comment:
Gotcha, that accounts for 2 of them, what about the other 2? since it is going from 7 to 3 on android for material 3. Just want to make sure we know what is changing here with material 3.